system_tests/conftest.py

```diff
@@ -41,10 +45,13 @@ def authorized_user_file():
     yield AUTHORIZED_USER_FILE
 
 
-@pytest.fixture
-def http_request():
+@pytest.fixture(params=['urllib3', 'requests'])
+def http_request(request):
```

Comment:
This is the py.test fixture request object. The parametrization passes in the params in this request object.